.github/workflows/ci.yaml

```diff
@@ -41,7 +42,6 @@ jobs:
     - uses: actions/checkout@v1
     - uses: reviewdog/action-misspell@v1
```

Comment:
```suggestion
    - uses: reviewdog/action-misspell@v1
      with:
        exclude: .github/workflows/ci.yaml
```

Reply:
I just made this change on my side.
FTR the PR is still a bit draft, but not a realt (Draft PR) since you need to have a real PR to test the GH actions properly.
So some additional fixes might come in.

Reply:
Sounds good, let me know when you'd like me to take another look. 👍